### PR TITLE
PXC-3752 Tasks to convert to Netlify (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Changes to be committed:
	modified:   Makefile
	new file:   netlify.toml
	new file:   requirements.txt
	new file:   runtime.txt
	modified:   source/add-node.rst
	modified:   source/bootstrap.rst
	new file:   source/conf-netlify/conf.py
	modified:   source/configure.rst
	new file:   source/diagnostics/libcoredumper.rst
	new file:   source/diagnostics/stacktrace.rst
	modified:   source/faq.rst
	modified:   source/features/highavailability.rst
	modified:   source/features/multimaster-replication.rst
	modified:   source/features/pxc-strict-mode.rst
	modified:   source/glossary.rst
	modified:   source/howtos/3nodesec2.rst
	modified:   source/howtos/centos_howto.rst
	modified:   source/howtos/crash-recovery.rst
	modified:   source/howtos/haproxy.rst
	modified:   source/howtos/proxysql-v2.rst
	modified:   source/howtos/proxysql.rst
	modified:   source/howtos/singlebox.rst
	modified:   source/howtos/ubuntu_howto.rst
	modified:   source/howtos/upgrade_guide.rst
	modified:   source/howtos/virt_sandbox.rst
	modified:   source/index.rst
	modified:   source/install/apt.rst
	modified:   source/install/compile.rst
	modified:   source/install/docker.rst
	modified:   source/install/index.rst
	modified:   source/install/tarball.rst
	modified:   source/install/yum.rst
	modified:   source/intro.rst
	modified:   source/limitation.rst
	modified:   source/management/data_at_rest_encryption.rst
	modified:   source/management/enforce_engine.rst
	modified:   source/manual/certification.rst
	modified:   source/manual/failover.rst
	modified:   source/manual/gcache_record-set_cache_difference.rst
	modified:   source/manual/monitoring.rst
	modified:   source/manual/state_snapshot_transfer.rst
	modified:   source/manual/threading_model.rst
	modified:   source/manual/xtrabackup_sst.rst
	modified:   source/overview.rst
	modified:   source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst
	modified:   source/security/encrypt-traffic.rst
	modified:   source/security/index.rst
	modified:   source/security/secure-network.rst
	modified:   source/verify.rst
	modified:   source/wsrep-files-index.rst
	modified:   source/wsrep-provider-index.rst
	modified:   source/wsrep-system-index.rst

### CODE REVIEW

File: doc/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+sphinx-autobuild==2020.9.1
```
Comment (nastena1606) [inline doc/requirements.txt:2]: Sphinx and sphinx-gitstamp packages are missing

File: doc/source/howtos/upgrade_guide.rst [hunk 2]
```diff
@@ -27,7 +27,7 @@ and variables mentioned in these documents when upgrading to |PXC| 8.0.
 
 .. _upgrade-guide-changed:
 
-Important changes in |pxc| |version|
+Important changes in PXC |version|
```
Comment (nastena1606) [inline doc/source/howtos/upgrade_guide.rst]: Do we use abbreviation in docs? If not, it should be fixed everywhere where it's not replaced

File: doc/source/wsrep-provider-index.rst [hunk 3]
```diff
@@ -532,6 +532,10 @@ the replica queue drops below :variable:`gcs.fc_factor` *
 
 .. variable:: gcs.fc_limit
 
+<<<<<<< HEAD
```
Comment (nastena1606) [inline doc/source/wsrep-provider-index.rst]: This has left from some merge and conflicts

File: doc/requirements.txt (hunk 1 above)
Comment (patrickbirch) [reply doc/requirements.txt:2]: Fixed.

File: doc/source/howtos/upgrade_guide.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/howtos/upgrade_guide.rst]: fixed.

File: doc/source/wsrep-provider-index.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/wsrep-provider-index.rst]: Fixed.

**Pull request merged**